README.md

```diff
@@ -1,30 +1,62 @@
-![GODOT](/logo.png)
+[![GODOT](/logo.png)](https://godotengine.org)
 
-https://godotengine.org
+## Godot Engine
 
-### The Engine
+Homepage: https://godotengine.org
 
-Godot is a fully featured, open source, MIT licensed, game engine. It focuses on having great tools, and a visual oriented workflow that can export to PC, Mobile and Web platforms with no hassle.
-The editor, language and APIs are feature rich, yet simple to learn, allowing you to become productive in a matter of hours.
+#### 2D and 3D cross-platform game engine
 
-### About
+Godot Engine is a feature-packed, cross-platform game engine to create 2D and
+3D games from a unified interface. It provides a comprehensive set of common
+tools, so that users can focus on making games without having to reinvent the
+wheel. Games can be exported in one click to a number of platforms, including
+the major desktop platforms (Linux, Mac OSX, Windows) as well as mobile
+(Android, iOS) and web-based (HTML5) platforms.
 
-Godot has been developed by Juan Linietsky and Ariel Manzur for several years, and was born as an in-house engine, used to publish several work-for-hire titles. Godot is a member project of the [Software Freedom Conservancy](https://sfconservancy.org)
```

Comment:
I find this historical part deserves to keep its place with the readme of the source code, because it is quite inspiring.

It shows that long time efforts, patience, friendship and passion can give birth to great projects, and can make dreams come true.

They could have kept the fruits of their hard works for themselves, but instead they choose to share it for free.
And thanks to their generosity and open-mindedness, it's the dreams of hundred of peoples around the world that have a chance to become true.

The growth of the community they have gathered could have inflated their egos, but looking at their comments and at the way they take decisions into the github shows that they always remained humble.

They don't even appear here : https://github.com/orgs/godotengine/people